python/triton/runtime/autotuner.py

```diff
@@ -163,6 +163,7 @@ def kernel_call():
         except (OutOfResources, CompileTimeAssertionFailure, PTXASError) as e:
             if verbose:
                 print(f"Autotuning failed with {e}")
+            e.__traceback__ = None
```

Comment:
The stack trace may still be helpful for users. I wonder if we can disable this optionally. 

Also wondering if it is possible to do this inside `filter_traceback`. Do we know what frames out of the stack holds a pointer to the output tensor?